cmd/registry/controller/parser.go

```diff
@@ -169,7 +171,10 @@ func generateCommand(action string, resourceName string) (string, error) {
 		case "version":
 			entityVal = resource.Version()
 		case "spec":
-			entityVal = resource.Spec()
+			entityVal = resource.SpecRevision()
```

Comment:
I thought we were not going to add a `SpecRevision` type and going to stick to only `Specs`

Reply:
Sorry, yes, this was done before we discussed not supporting SpecRevisions in the controller... which now it looks like we're going to discuss next week? I imagine I should leave this alone until then.